### DISCUSSION TITLE
Chromium Versions

### DISCUSSION THREAD

**Discussion by Billyjamesdoh** [SafeExamBrowser/seb-win-refactoring#460] [Q&A]
SEB states from: Release Notes [Safe Exam Browser for Windows] Version 3.4.0
Updated browser engine to version 103.0.12 (chromium version 103.0.5060.134).

Can we use newer releases? Our security policy states we have to be on the latest release 105.0.5195.102 or 107.0.5277.0 for our systems

can anyone help?

**Comment by dbuechel** [comment]
No, that's unfortunately not possible. But we always update the engine to the latest available version for every release.

**Reply by Billyjamesdoh** [reply]
Would you know what the security risks are of running the outdated version?

Would it be all the security issue and bugs that are found-- upto the new release?

also what is the reason for not keeping up to date?

many thanks 

Billy

**Reply by dbuechel** [reply]
We'll certainly make a dedicated release to update the engine if there is a critical security issue. Apart from that, we simply don't have the resources to keep up with the release cycle of the browser engine.
